Remotion/Development/UnitTesting.Compilation/Development.UnitTesting.Compilation.csproj

```diff
@@ -14,7 +14,7 @@
     <ProjectReference Include="..\..\Core\Core\Core.Core.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="System.CodeDom" Version="7.0.0" />
+    <PackageReference Include="System.CodeDom" />
```

Comment:
This changes the dependency to version 8.0.0 - is this intentional?

Reply:
Yes I think this was missed during a previous dependency upgrade as some dependencies of `System.CodeDom` were already on package version `8.0.0`.